README.md

```diff
@@ -531,11 +535,11 @@ A few sample addons listed below. Please do not hesitate to add your own addon t
 
 Note: The addons under [https://github.com/otobus](https://github.com/otobus) organization implemented as a sample, but feel free to use them in your project with respecting their licenses.
 
-## License
+## Copyright and License
 
 MIT
 
-Copyright (c) 2019 Mustafa Turan
+Copyright (c) 2017 Mustafa Turan
```

Comment:
2022

Reply:
Should this be the starting year instead of latest year?

Reply:
I think on every version it should change(I am planning to bump the version at 2022). IMO it should sync with the published date of the version.